packages/suite-data/files/translations/fr.json

```diff
@@ -381,7 +381,7 @@
   "TR_BUY_DETAIL_PENDING_SUPPORT": "Accéder à la page d’assistance du fournisseur",
   "TR_BUY_DETAIL_PENDING_TITLE": "Traitement de votre transaction en cours...",
   "TR_BUY_DETAIL_SUBMITTED_GATE": "Aller au portail de paiement",
-  "TR_BUY_DETAIL_SUBMITTED_TEXT": "Veuillez cliquer sur le bouton ci-dessous pour terminer de saisir vos données sur le site du fournisseur.",
+  "TR_BUY_DETAIL_SUBMITTED_TEXT": "Veuillez cliquer sur le bouton ci-dessous pour terminer de saisir\n          vos données sur le site du fournisseur.",
```

Comment:
@MiroslavProchazka

Reply:
No idea how this keeps happening, I deleted the translation and approved a previous one.